src/wopiserver.py

```diff
@@ -619,6 +621,21 @@ def cboxDownload_deprecated():
     return iopDownload()
 
 
+def initEncryptionKey(cls):
+    try:
+        with open(cls.config.get('security', 'jwesecretfile'), 'r') as kf:
```

Comment:
A minor comment: for the secret we can safely reuse the already configured `wopisecretfile` value, which is only used by the wopiserver (not shared with Reva) to encode the JWT. In other words the same secret either encodes or encrypts depending on the config flag.

Reply:
It needs to be an AES256 keywrap. 

`{"k":"P6mA6K6Jsaob6-ch7krZTacV9VRE7Xx58bZ8vbkV6Lw","kty":"oct"}`

Reply:
Let me check again. I tried it with a plain "key_from_password" approach, but the byte length was always mismatching.